Service/TriggerdialogService.php

```diff
@@ -137,7 +137,7 @@ public function updateCampaignVariable(TriggerCampaign $triggerCampaign)
         $xml = new \SimpleXMLElement('<updateCampaignVariableRequest xmlns:ns2="urn:pep-dpdhl-com:triggerdialog/campaign/v_10"></updateCampaignVariableRequest>');
         $this->transformData($xml, $data);
 
-        $response = $this->client->request('POST', '/rest-mas/campaign/variable/', ['body' => $xml->asXML()]);
+        $response = $this->client->request('POST', '/rest-mas/variabledefinitions/', ['body' => $xml->asXML()]);
```

Comment:
Is this change breaking?
Anyhow, you should consider to update the version number of this plugin.

Reply:
they changed the two route in the old api, so this will be necessary even for old clients